js/data/data_source/data_source.js

```diff
@@ -190,7 +190,7 @@ var DataSource = Class.inherit({
         /**
         * @name DataSourceOptions_store
         * @publicName store
-        * @type Store|Array<any>|Object
+        * @type Store|StoreOptions|Array<any>|Object|string|any
```

Comment:
What's the case for `string`?

Reply:
url ?

Reply:
Data Source can be [specified as an URL](https://js.devexpress.com/Documentation/Guide/Data_Layer/Data_Layer/#Creating_DataSource/From_url), Store [cannot](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/DataSource/Configuration/store/).